po/uk.po

```diff
@@ -1004,23 +995,23 @@ msgstr "Архів обкладинки"
 
 #: picard/coverart/providers/caa.py:184
 msgid "Cover Art Archive: Release"
-msgstr "Архів обкладинок: реліз"
+msgstr "Архів обкладинок: Реліз"
 
 #: picard/coverart/providers/caa_release_group.py:44
 msgid "Cover Art Archive: Release Group"
-msgstr "Архів обкладинок: група релізів"
+msgstr "Архів обкладинок: Release Group"
```

Comment:
Sure about this? Most languages find some kind of translation for this expression.